resources/js/Components/MismatchRow.vue

```diff
@@ -200,12 +197,16 @@ export default defineComponent({
 <style lang="scss">
 @import '~@wmde/wikit-tokens/dist/_variables.scss';
 
-    .wikit-Link.break-line-link {
-      width: 100%;
-    }
-    .wikit-Link__content {
+  a {
+    &:__content {
       word-break: break-word;
     }
+
+    &:break-line-link {
+      width: 100%
+    }
+  }
+
```

Comment:
When using `wikit-link`, it has its own elements inside (a `span` nested in `a` element), these selectors are for them.  I suppose, for our new case something like below should be enough. 

```sass
a {
  &.break-line-link {}
}
```

Note: This is just for demonstration, I didn't add styling on purpose.

Reply:
Thanks for the hint! I left only the `break-line-link` class, as the other one is redundant as you commented.